style.css

```diff
@@ -0,0 +1,3 @@
+.Intro {
+  color: aqua;
+}
```

Comment:
- [ ] According to the project requirement, kindly make your css file one class with styles for your header for proper accessibility.

Reply:
@ademibowale I don't fully understand what you mean by making the css file one class